zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/schema/SchemaManagerIT.java

```diff
@@ -51,13 +46,41 @@
 @Testcontainers
 public class SchemaManagerIT {
 
-  private static final ObjectMapper MAPPER = new ObjectMapper();
   private static ExporterConfiguration config = new ExporterConfiguration();
+  private static ElasticsearchClient elsClient;
+  private static OpenSearchClient osClient;
+
+  @Container
+  private static final ElasticsearchContainer ELS_CONTAINER =
+      TestSupport.createDefeaultElasticsearchContainer();
+
+  @Container
+  private static final OpensearchContainer<?> OPENSEARCH_CONTAINER =
+      TestSupport.createDefaultOpensearchContainer();
+
   private IndexDescriptor index;
   private IndexTemplateDescriptor indexTemplate;
 
+  @BeforeAll
+  public static void init() {
+    config.getConnect().setUrl(ELS_CONTAINER.getHttpHostAddress());
+    elsClient = new ElasticsearchConnector(config.getConnect()).createClient();
+
+    config.getConnect().setUrl(OPENSEARCH_CONTAINER.getHttpHostAddress());
+    osClient = new OpensearchConnector(config.getConnect()).createClient();
+  }
+
   @BeforeEach
-  public void refresh() throws IOException {
+  public void refresh(final TestInfo testInfo) throws IOException {
+    if (testInfo.getDisplayName().contains("elasticsearch")) {
+      elsClient.indices().delete(req -> req.index("*"));
+      elsClient.indices().deleteIndexTemplate(req -> req.name("*"));
+    } else if (testInfo.getDisplayName().contains("opensearch")) {
+      osClient.indices().delete(req -> req.index(config.getIndex().getPrefix() + "*"));
+      osClient.indices().deleteIndexTemplate(req -> req.name("*"));
+    }
```

Comment:
💭 This does the job indeed, good thinking! But I'm also afraid of relying on the test method display name, it feels like that's easily breakable.
I don't have a better idea right now, I'm just thinking out loud.. 

💭 Maybe the only other idea is to add the `connectionType` as a parameter and use `testInfo.getTestMethod() -> parameters` to make this decision

Reply:
I don't like adding a parameter we don't use for tests, I'll just use named arguments insteada